modules/installation-preparing-restricted-cluster-to-gather-support-data.adoc

```diff
@@ -9,9 +9,16 @@ Clusters using a restricted network must import the default must-gather image in
 
 .Procedure
 
-. Import the default must-gather image from your installation payload:
+* Import the default must-gather image from your installation payload:
 +
 [source,terminal]
 ----
 $ oc import-image is/must-gather -n openshift
 ----
+
+When running the `oc must-gather` command, use the `--image` flag as part of must-gather and point to the payload image, as in the following example:
++
```

Comment:
The "+" here should be removed - it's rendering when you look at the built version